db/schema.rb

```diff
@@ -200,13 +200,14 @@
     t.datetime "updated_at",            null: false
     t.string   "inscription_reference"
     t.date     "inscription_date"
-    t.string   "entity_type"
     t.string   "neighbourhood"
     t.string   "district"
     t.string   "scope"
     t.integer  "associations_count"
     t.integer  "members_count"
     t.string   "approach"
+    t.boolean  "invalidate"
```

Comment:
This new `invalidate` field is not present at the migration file 🤔  maybe you're changing between branches and a swift

Reply:
is from other branch, it doesn't belongs here....